### PR TITLE
Fix fonts google

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- configuration of google fonts through a link added in html (head)
- corrections

- Fonts:
Montserrat
Poppins
Roboto
Windsong

### CODE REVIEW

File: app/components/projects/galleries/complementary.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const COMPLEMENTARY = [
     href: "#complementar",
     title: "Consultoria Complementar",
     category1: "concept board",
-    category2: "Sala",
+    category2: "  Cozinha",
```
Comment (doublethink13) [inline app/components/projects/galleries/complementary.js:25]: is the extra space needed?

File: app/styles/global.css [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
-@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,100&display=swap");
-/* Roboto thin */
-@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap");
-/* Windsogn regular 400 */
-@import url("https://fonts.googleapis.com/css2?family=WindSong&display=swap");
-
 @import "~slick-carousel/slick/slick.css";
 @import "~slick-carousel/slick/slick-theme.css";
 
+/* font-family: 'Montserrat', sans-serif;
```
Comment (doublethink13) [inline app/styles/global.css:4]: comments

File: app/components/projects/galleries/complementary.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/components/projects/galleries/complementary.js]: no, well noted! Thank you.
Extra space was deleted.

File: app/styles/global.css (hunk 2 above)
Comment (Joanartdacosta) [reply app/styles/global.css]: comments deleted!

**Pull request merged**